.pipelines/.vsts-vhd-builder.yaml

```diff
@@ -104,3 +104,18 @@ stages:
       LOCATION: $(PACKER_BUILD_LOCATION)
     jobs:
       - template: ./templates/e2e-template.yaml
+  - stage: Generate and Publish Testdata
+    condition: succeeded()
+    jobs:
+      - job: Generate_Publish_Testdata
+        timeoutInMinutes: 10
+        steps:
+        - bash: |
+            make generate
+          displayName: Generate Testdata
```

Comment:
I think this is already the case, but probably worth double checking that the checkin build will be automatically triggered whenever we end up changing template content